packages/core/src/RenderingEngine/WSIViewport.ts

```diff
@@ -228,8 +234,12 @@ class WSIViewport extends Viewport {
     this.setProperties({});
   }
 
-  protected getScalarData() {
-    return null;
+  protected _getScalarData() {
+    // Return an empty CanvasScalarData object
+    const emptyData = new Uint8ClampedArray() as CanvasScalarData;
+    emptyData.getRange = () => [0, 255];
+    emptyData.frameNumber = -1;
```

Comment:
Can you add an emptyData.region that is a rectangle of the enclosing region, and a scale parameter?  The region should be zero sized for now, and the scale can be 1, but again I'd like to start thinking about the future.